astroquery/jplspec/core.py

```diff
@@ -0,0 +1,253 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import os
+import warnings
+
+import astropy.units as u
+from astropy.io import ascii
+
+# all Query classes should inherit from BaseQuery.
+from ..query import BaseQuery
+# async_to_sync generates the relevant query tools from _async methods
```

Comment:
these comments are from the template and are intended to be instructional; could you remove them?  I leave to your discretion which ones ought to be removed